### ISSUE TITLE
Mention in README how much time building the glasses take

### ISSUE THREAD

**Issue by mickaelistria** [mobicarton/guide-template#2]
In the README.md, it would be interesting to mention what's the average time to build the glasses for a n00b (I think I've read about 2 hours in the paper).

**Comment by makowildcat** [comment]
Yes, perfect, I will add it (indeed it is around 2 hours for the first time, and quickly down to 45/50 minutes after making a bunch of them... at least for this version)
